StructureDefinition/NHSDigital-MessageHeader.StructureDefinition.xml

```diff
@@ -53,12 +69,6 @@
       </type>
       <mustSupport value="true" />
       <binding>
-        <extension url="http://hl7.org/fhir/StructureDefinition/elementdefinition-bindingName">
```

Comment:
Just a comment, not blocking this change. Are being used in any implementation?  It would be good to know what the use case for these extensions were and why they're no longer wanted.  It's potentially a breaking change if these were used in any way (the extensions), but personally I'd have a preference to do without them, but that would be without knowing what the initial use case(s) was/were...

Reply:
The surrounding description relates to EPS activities. I will document along with issues for second level review

Reply:
As far as I can tell, this isn't removing an extension created by us, but one automatically added by Forge "bindingName" (I believe in older versions of Forge). This only has an effect on the rendering on simplifier (giving an alias to the binding) and doesn't have any effect on validation against the StructureDefinition itself. Happy for this to go through.